### ISSUE TITLE
When renaming Column in a Table SQLiteStudio hangs or freezes

### ISSUE THREAD

**Issue by csanyipal** [pawelsalawa/sqlitestudio#3728]
### Details
When I want to rename a Column and hit the green Commit button SQLiteStudio freezes and do nothing.

### Steps to reproduce
On the Structure tab of a Table I double click on a Column name.
The dialog window opens.
I change there the name of that column and hit OK.
Then hit on the green button to commit changes.
SQLiteStudio freezes.

### Operating system
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.4 LTS
Release:	18.04
Codename:	bionic

### SQLiteStudio version
v3.2.1

**Comment by pawelsalawa** [comment]
Is it a big table (hundreds of thousands of rows or more)?

**Comment by csanyipal** [comment]
>     1. Is it a big table (hundreds of thousands of rows or more)?

No, it happens regardless of the size of the table. I tried it out on a table with ten rows only and on table with hundreds of rows in this database.

>     2. Does it happen for other tables in this database?

Yes, it does happen for other tables in this database.

>     3. Is this database being used by other application at the time you're changing this column?

No.

>     4. Is database placed on some network share?

No, it is not placed out there.

**Comment by pawelsalawa** [comment]
I see. Could you please create a fresh new database and table there and try repeating the same?

**Comment by csanyipal** [comment]
> I see. Could you please create a fresh new database and only a single table there and try repeating the same?

I just created a fresh new database with only a single table there and try repeating the same out there.

No freezes this time.

**Comment by pawelsalawa** [comment]
Okay, this looks like you have depencencies which cause this bug.

What other views (in their queries) & tables (in their foreign keys) are dependant on this problematic table?

**Comment by csanyipal** [comment]
> Okay, this looks like you have dependencies which cause this bug.
> 
> What other views (in their queries) & tables (in their foreign keys) are dependant on this problematic table?
> 
> You can find this out easily by doing your column modification and then not execute it, but just review the DDL to be executed (if you disabled these preview, you can restore them in configuration).

I just unset in Configuration / General the 'Don't show DDL preview dialog when committing schema changes.
But when I edit and modify a Column name and hit on the Commit green button SQLiteStudio freezes again but do not show the DDL dialog before. Why?
